docs/src/theme/faq.module.css

```diff
@@ -0,0 +1,83 @@
+.pills,
```

Comment:
@jfcurran could you help @k9ert @vinckr styling the FAQ a bit? I think the tags need some love :) To run the project, check it out - use the GH command line `gh `:

```
$ git clone ... kratos ...
$ cd path/to/kratos
$ gh pr checkout 1039
$ cd docs
$ npm i
$ cd ..
$ node ./docs/scripts/gen-faq.js
$ npm start
```

Then navigate to http://localhost:3000/kratos/docs/next/faq

The page looks like so:

<img width="1392" alt="Bildschirmfoto 2021-02-25 um 10 14 30" src="https://user-images.githubusercontent.com/3372410/109130651-4561e800-7752-11eb-9a2d-7f7ea590f337.png">